phinx.php

```diff
@@ -15,7 +15,7 @@
 
 require_once __DIR__ . '/vendor/autoload.php';
 
-$kernel = new Kernel((string) Environment::fromServer($_SERVER), false);
+$kernel = new Kernel((string) Environment::fromServer(\getenv()), false);
```

Comment:
@smoench 

I think we need to set an environment variable in [`.travis.yml`](https://github.com/opencfp/opencfp/blob/4430b2d56bd9f2c1640e313d9c104f731a2ae406/.travis.yml#L6), now, so we can properly determine the environment on Travis.

Can you adjust, please?

It's already set in [`Makefile`](https://github.com/opencfp/opencfp/blob/4430b2d56bd9f2c1640e313d9c104f731a2ae406/Makefile#L22).